src/envs/behavior_options.py

```diff
@@ -0,0 +1,1104 @@
+"""Hardcoded options for BehaviorEnv.
```

Comment:
in the spirit of smaller reviews and faster iteration, I'm going to skip this file for now and review it on my next pass through, after my first round comments have been addressed!